### PR TITLE
Juicy-Jaguars

### PR DESCRIPTION
**Pull request by DanGill**

### CODE REVIEW

**Comment by SebastiaanZ** [comment]
The last two commits to fix the linting were made after the deadline, although it's only one and two minutes over respectively.

No merge issues

No flake8 linting errors (but note the two commits after the deadline)

File: juicy-jaguars/README.md [hunk 1]
```diff
@@ -1,2 +1,83 @@
-# Juicy Jaguars
-This is the default README of your team's project. Please replace this by a README with more information on your project. At the very least, your README should contain information on how to set-up and run your project.
+<div align="center">
```
Comment (eivl) [inline juicy-jaguars/README.md:1]: great README

File: juicy-jaguars/Web95/Web95/html_parse.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    """Class used to parse HTML."""
+
+    def __init__(self, html, basedir, request):
+        """Store HTML, setup Soup."""
+        self.html = html
+        self.soup = BeautifulSoup(html, "html.parser")
+        self.basedir = basedir
+        self.request = request
+
+    def parse(self):
```
Comment (eivl) [inline juicy-jaguars/Web95/Web95/html_parse.py:18]: This is very clever

File: juicy-jaguars/Web95/Web95/views.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+import os  # Various functions
+import urllib.parse  # For parsing URLs with escape codes
+import requests  # For fetching webpages
+from math import floor  # For image resizing
+from PIL import Image, UnidentifiedImageError  # For image processing
+from .html_parse import HtmlParser  # Our HTML parser
+from .settings import BASE_DIR  # For Base Directory path
+
+
+def landing_page(request):
```
Comment (eivl) [inline juicy-jaguars/Web95/Web95/views.py:16]: looks solid. I like the easter eggs.

File: juicy-jaguars/setup_beta.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
+"""Beta Setup script for Web95."""
+# ========= BETA VESION =========
+
+import os
+import sys
+import subprocess
+subprocess.call(["pip3", "install", "scprint"], stdout=open(os.devnull, 'w'))
```
Comment (eivl) [inline juicy-jaguars/setup_beta.py:7]: might be better to just have the module locally.

Comment (eivl) [reply juicy-jaguars/setup_beta.py:7]: well.... guess its fine. but it does feel a bit hacky.

**Review by eivl — COMMENTED** [review]
What are the licencing on the icon and fonts used?

**Pull request merged**